### ISSUE TITLE
Improve documentation for DRAG_LAW

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#6258]
Jason,
I was just trying to answer a question about the way drag laws are implemented and I found the discussion in the user guide scattered and hard to follow.  Could you add a table with the possible values of DRAW_LAW (I assume 'sphere', 'cylinder', 'screen') with a pointer to where the laws are actually written out (tech guide).  Thanks.

**Comment by btrettel** [comment]
There's also 'POROUS MEDIA' drag. Because I had been figuring this out myself, I made a pull request with a partial table, commented out for now. See pull request #6261. I would have completed the table and added the appropriate drag equations for spheres and cylinders, but I do not have the Holman textbook and am not certain if the 1 < Re < 1000 sphere drag equation is as printed in that book (see the comments in func.f90, line 2657).

The code is a little scattered on this too, with the DRAG subroutine in func.f90 rather than part.f90. Though this was easy enough to locate.

**Comment by rmcdermo** [comment]
Thanks, Ben.  I made minor edits, but this is done in PR #6264.
